main/streams/streams.c

```diff
@@ -1406,6 +1406,11 @@ PHPAPI int _php_stream_set_option(php_stream *stream, int option, int value, voi
 	return ret;
 }
 
+PHPAPI int _php_stream_sync(php_stream *stream)
+{
+	return php_stream_set_option(stream, PHP_STREAM_OPTION_SYNC_API, PHP_STREAM_SYNC_FSYNC, NULL);
```

Comment:
ditto here

Reply:
@derickr I thought when I first saw your comments that my IDE was replacing tabs with spaces because that would be my normal set-up, but actually looks like it isn't and https://github.com/php/php-src/blob/b8a54b1d23461ea6c863d2fdf4297b8c4d911181/main/streams/streams.c#L1411 is showing the correct tab separation. Any ideas? Perhaps this is just the diff displaying weirdly?